src/Targets/Microsoft.FSharp.DesignTime.targets

```diff
@@ -34,9 +34,24 @@
           >
 
     <ItemGroup>
+      <_CompilerCommandLineArgs Include="@(FscCommandLineArgs, ';');"/> 
+    </ItemGroup>
+
+  </Target>
+
+
+<!-- Saved
```

Comment:
We use history to save code, instead of commenting out code. Or point to a bug indicating where we're going to change it.